src/debugpy/server/cli.py

```diff
@@ -294,14 +313,9 @@ def run_module():
     # On Python 2, module name must be a non-Unicode string, because it ends up
     # a part of module's __package__, and Python will refuse to run the module
     # if __package__ is Unicode.
-    target = (
-        compat.filename_bytes(options.target)
-        if sys.version_info < (3,)
-        else options.target
-    )
 
     log.describe_environment("Pre-launch environment:")
-    log.info("Running module {0!r}", target)
+    log.info("Running module {0!r}", options.target)  # Log must be unicode.
```

Comment:
`{!r}` should take care of necessary quoting and escaping if it's a byte string, no? OTOH we can see the final encoding result that gets passed to `runpy`, which may be useful for diagnosing encoding issues.

Reply:
You're right that the `!r` makes it work (instead of throwing a `UnicodeDecodeError` as would be the case if it wasn't there), but I believe that the output is nicer if both are unicode in the case where the target actually has unicode characters.